chaos_genius/views/data_source_view.py

```diff
@@ -282,18 +282,19 @@ def create_data_source():
             db_connection_uri = create_sqlalchemy_uri(**db_config)
         else:
             sourceRecord = sourceCreationPayload
-            db_mapper = DATABASE_CONFIG_MAPPER[source_form.get("sourceDefinitionId")]
-            input_configuration = source_form.get("connectionConfiguration")
-            if db_mapper["db_type"] in ["mysql", "postgres"]:
-                db_config = {
-                    "host": input_configuration[db_mapper["host"]],
-                    "port": input_configuration[db_mapper["port"]],
-                    "database": input_configuration[db_mapper["database"]],
-                    "username": input_configuration[db_mapper["username"]],
-                    "password": input_configuration[db_mapper["password"]],
-                    "db_type": db_mapper["db_type"],
-                }
-                db_connection_uri = create_sqlalchemy_uri(**db_config)
+            # TODO: delete this
```

Comment:
Can you check the implication of removing this code? Let's delete the commented code too.

Reply:
@manassolanki you only removed this section. Plus I don't have enough context for this API call to check the implications.